massa-consensus-exports/Cargo.toml

```diff
@@ -8,17 +8,14 @@ edition = "2021"
 testing = ["massa_models/testing", "massa_execution_exports/testing", "massa_pool_exports/testing", "massa_pos_exports/testing", "massa_protocol_exports/testing", "massa_storage/testing", "dep:mockall"]
 
 [dependencies]
-crossbeam-channel = {workspace = true}   # BOM UPGRADE     Revert to "0.5.6" if problem
 displaydoc = {workspace = true}
 nom = {workspace = true}
 serde = {workspace = true, "features" = ["derive"]}
 serde_json = {workspace = true}   # BOM UPGRADE     Revert to "1.0" if problem
 thiserror = {workspace = true}
-jsonrpsee = {workspace = true, "features" = ["server"]}
 tokio = {workspace = true, "features" = ["sync"]}
 mockall = {workspace = true, "optional" = true}   # BOM UPGRADE     Revert to {"version": "0.11.4", "optional": true} if problem
 massa_channel = {workspace = true}
-massa_hash = {workspace = true}
```

Comment:
I think there are check comments associated (see at the bottom of the diff page on github).
`massa_hash` is used in the doc comments examples (https://github.com/massalabs/massa/blob/main/massa-consensus-exports/src/export_active_block.rs#L144), so I think we can put it as [dev-dependencies] instead?

Can you check for similar behaviours?

Reply:
I'm agree, we have to check all combinations.

Reply:
It seems that the deps analyzer don't recognize the doctests for example